src/hooks/useArNSContract/useArNSContract.tsx

```diff
@@ -36,7 +37,7 @@ export default function useArNSContract() {
       setTimeout(() => {
         // short delay to prevent another dispatch
         setSendingContractState(false);
-      }, 100);
+      }, 60000);
```

Comment:
i'm not experiencing any issues with state loading, might need to add some logs here to see why it would be dispatching

Reply:
also one minute is far too long.